app/dashboard/views.py

```diff
@@ -2122,6 +2122,7 @@ def profile(request, handle):
 
             return JsonResponse(msg, status=msg.get('status', 200))
     context['show_activity'] = request.GET.get('p', False) != False
+    context['theme_type'] = request.user.profile.custom_theme if request.user.is_authenticated else 'light'
```

Comment:
you can lose `if request.user.is_authenticated else 'light'`

Reply:
I can remove the if condition. When it's not set, it falls to default theme

Reply:
Removing this will cause error anytime a user is not authenticated and is viewing a profile